account_ecotax/models/account_move_line.py

```diff
@@ -0,0 +1,119 @@
+# © 2014-2023 Akretion (http://www.akretion.com)
+#   @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import Command, api, fields, models
+
+
+class AcountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    ecotax_line_ids = fields.One2many(
+        "account.move.line.ecotax",
+        "account_move_line_id",
+        string="Ecotax lines",
+        copy=True,
+    )
+    subtotal_ecotax = fields.Float(
+        string="Ecotax", digits="Ecotax", store=True, compute="_compute_ecotax"
+    )
+    ecotax_amount_unit = fields.Float(
+        digits="Ecotax",
+        string="Ecotax Unit",
```

Comment:
This name is confusing. Is it a unit price?

Reply:
Yes